lib/Driver/Driver.cpp

```diff
@@ -1580,8 +1618,9 @@ void Driver::buildJobs(const ActionList &Actions, const OutputInfo &OI,
   }
 
   for (const Action *A : Actions) {
-    (void)buildJobsForAction(C, cast<JobAction>(A), OI, OFM, TC,
-                             /*TopLevel*/true, JobCache);
+    if (isa<JobAction>(A))
```

Comment:
(I also generally add braces when the if-body is multiple lines even if it's still just one statement. I'm not such a fan of LLVM's brace-less rule.)